prow/scripts/cluster-integration/helpers/fluent-bit-stackdriver-logging.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# prepare stackdriver logging integration
+# - creates secret which holds ServiceAccount with LogWritter permission (for Stackdriver access)
+# - creates installation override for fluent-fit
+function prepare_stackdriver_logging() {
+    local filepath="${1}"
+    if [[ -z "${filepath}" ]]; then
+        echo "Filepath for installation override not given"
+        return 1
+    fi
+
+    # create secret
+    echo "creating stackdriver secret"
+    kubectl create namespace kyma-system || true # in case the namespace already exists
```

Comment:
Namespace creation may fail due to RBAC or any other reason. Piping to true will cover that up. Can we be more explicit by checking the existence of the namespace?